CHANGELOG.md

```diff
@@ -9,6 +9,32 @@ Looking for changes that affect our C API? See the [C API Changelog](lib/c-api/C
 
 ## **Unreleased**
 
+## 4.3.0 - 10/05/2024

```

Comment:
Summary of changes please!

Reply:
I disabled auto-merge, it's fine. I'll enable it after I added the manual commits.
We first update `0.x` crates last time and then update the change log.
I'll do it in that order.

Reply:
The order doesn't really matter. CI will need to rerun after you push the changes, so push it all in one go

Reply:
I prefer to do it in separate steps for now. I'll do it in one go when I'm more confident.